test/shell/test_shell.py

```diff
@@ -112,6 +125,7 @@ def test_train_shell(train_env: TrainEnv, caplog) -> None:
             assert line.endswith("270.0")
 
 
+@pytest.mark.parametrize("listify_dataset", [True])
```

Comment:
I guess we would want to test `True` and `False`.

Reply:
I can add the `False` cases to all the tests as well, but I added a separate test `test_listify_dataset` that tests both `True` and `False`.